meadServer/client/src/App.js

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
For the CSV export, seems like you can list your fields: https://www.npmjs.com/package/json2csv
Instead of specifying it each time in the hash